gph/python/test/test_ripser.py

```diff
@@ -1,14 +1,13 @@
 import numpy as np
 import pytest
+from gph import ripser_parallel as ripser
```

Comment:
Should we not separate the import of the package being tested by the general import of externals libraries ?

Reply:
Well, it sort them alphabetically, the issue for me is that we cannot differentiate current library import, or "internals" imports from external dependencies import.
I think it is important to have a clear difference between both

Reply:
Makes sense!  I'll revert.

Reply:
You can resolve the conversation afterwards, thank you !

Reply:
Done.